.devcontainer/devcontainer.json

```diff
@@ -47,6 +47,6 @@
   "remoteUser": "node",
 
   // Lifecycle commands
-  "postCreateCommand": "echo 'eval \"$(mise activate zsh)\"' >> ~/.zshrc && mise trust && mise install && uv tool install specify-cli --from git+https://github.com/github/spec-kit.git && cargo install cargo-release git-cliff",
+  "postCreateCommand": "echo 'eval \"$(mise activate zsh)\"' >> ~/.zshrc && mise trust && mise install && uv tool install specify-cli --from git+https://github.com/github/spec-kit.git && . ~/.cargo/env && cargo install cargo-release git-cliff",
```

Comment:
1. When the devcontainer is first built for a github codespace, is `. ~/.cargo/env` guaranteed to exist? I think not.
2. My preference is to create a dedicated script which is called, rather than stuff all these commands into one line. please do that.